osbuild-composer.spec

```diff
@@ -9,7 +9,7 @@
 
 %global goipath         github.com/osbuild/osbuild-composer
 
-Version:        46
```

Comment:
FWIW, I find this versioning weird. It would be problematic if we had to have different backports for RHEL-8 and RHEL-9. In this case, it either does not make sense to make a release, but just add downstream patches to the RHEL dist-git. Or we should reflect the target release version in the osbuild version (something like the release part in RPM NVR).

Reply:
Agreed, let's discuss it at some point, it's indeed quite weird. I'm afraid though that we don't have time for the discussion now, so let's just go with it, we can surely adjust it if some fixes are needed for z-stream.